src/components/MedicalServiceMasterPanel.js

```diff
@@ -98,6 +99,17 @@ class MedicalServiceMasterPanel extends FormPanel {
               onChange={(p) => this.updateAttribute("level", p)}
             />
           </Grid>
+          <Grid item xs={2} className={classes.item}>
+            <NumberInput
+              min={0}
+              module="admin"
+              label="medical.service.maximumAmount"
+              name="maximumAmount"
+              readOnly={readOnly}
+              value={edited && edited.maximumAmount ? edited.maximumAmount : ""}
```

Comment:
Using optional chaining here will make this value line shorter:
`value={edited?.maximumAmount ?? ""}`